### PR TITLE
Smaato: add gpp and gpp_sid into json to ensure it is being sent

### PR DESCRIPTION
**Pull request by Enigo**
Added a test for Smaato adapter to ensure that gpp data is being sent when set

### CODE REVIEW

**Review by onkarvhanumante — DISMISSED** [review]

File: adapters/smaato/smaatotest/exemplary/simple-banner.json [hunk 1]
```diff
@@ -118,7 +120,9 @@
             "coppa": 1,
             "ext": {
               "gdpr": 1,
-              "us_privacy": "uspConsentString"
+              "us_privacy": "uspConsentString",
+              "gpp": "gppString",
+              "gpp_sid": [7]
```
Comment (gargcreation1992) [inline adapters/smaato/smaatotest/exemplary/simple-banner.json:125]: Could we also add `gpp` and `gpp_sid` json for other media types as well like video and native?

Comment (Enigo) [reply adapters/smaato/smaatotest/exemplary/simple-banner.json:125]: We could, but why?

Comment (gargcreation1992) [reply adapters/smaato/smaatotest/exemplary/simple-banner.json:125]: To cover the test case for other media types.

Comment (Enigo) [reply adapters/smaato/smaatotest/exemplary/simple-banner.json:125]: alright, pushed

Comment (gargcreation1992) [reply adapters/smaato/smaatotest/exemplary/simple-banner.json:125]: thanks @Enigo . LGTM now!

**Review by gargcreation1992 — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**